sql/moz-fx-data-shared-prod/external_derived/imf_exchange_rates_v1/query.py

```diff
@@ -16,7 +16,6 @@
     "CA",
     "CN",
     "CO",
-    "GB",
```

Comment:
I'd suggest sorting this list of country codes alphabetically so it's easier to read, which would have the side benefit of making it easier to notice potential duplicates.